STACpopulator/implementations/HRDPS_CRIM/collection_config.yml

```diff
@@ -0,0 +1,17 @@
+title: HRDPS
+id: HRDPS_CRIM
+description: High Resolution Deterministic Prediction System
+keywords: ['HRDPS', 'Climate Change']
+license: "CC-BY-4.0"
+spatialextent: [-180, -90, 180, 90]
+temporalextent: ['1850-01-01', '2500-01-01']
```

Comment:
Adjust according to https://open.canada.ca/data/api/action/package_show?id=5b401fa0-6c29-57f0-b3d5-749f301d829d

There is also some more metadata in there that might be relevant (eg: contact info, keywords, etc.)